tests/lib/openstack_query/test_query_executer.py

```diff
@@ -1,51 +1,93 @@
 from unittest.mock import MagicMock, patch
 import pytest
+
+from enums.cloud_domains import CloudDomains
 from openstack_query.query_executer import QueryExecuter
 from tests.lib.openstack_query.mocks.mocked_props import MockProperties
 
-# pylint:disable=protected-access
-
 
 @pytest.fixture(name="instance")
 def instance_fixture():
     """
     Returns an instance with a mocked runner and prop enum class
     """
     mock_prop_enum_cls = MockProperties
-    mock_runner = MagicMock()
-    return QueryExecuter(mock_prop_enum_cls, mock_runner)
+    mock_runner_cls = MagicMock()
+    return QueryExecuter(mock_prop_enum_cls, mock_runner_cls)
 
 
-def test_set_filters(instance):
+def test_set_client_side_filter_func(instance):
     """
-    Tests that set_filters setter method works as expected
+    Tests that client_side_filter_func setter method works as expected
     """
-
     mock_client_filter = MagicMock()
-    mock_server_side_filter = MagicMock()
-    instance.set_filters(mock_client_filter, mock_server_side_filter)
+    instance.client_side_filter_func = mock_client_filter
     assert instance._client_side_filter_func == mock_client_filter
-    assert instance._server_side_filters == mock_server_side_filter
+
+
+def test_get_client_side_filter_func(instance):
+    """
+    Tests that client_side_filter_func getter method works as expected
+    """
+    mock_client_filter = MagicMock()
+    instance._client_side_filter_func = mock_client_filter
+    assert instance.client_side_filter_func == mock_client_filter
+
+
+def test_set_server_side_filters(instance):
+    """
+    Tests that server_side_filters setter method works as expected
+    """
+    mock_client_filter = MagicMock()
+    instance.server_side_filters = mock_client_filter
+    assert instance._server_side_filters == mock_client_filter
```

Comment:
Ditto on my previous comment, just test a property instead of a getter / setter:

```
def test_client_side_filter_prop(instance):
    mock_client_filter = MagicMock()
    instance.client_side_filter_func = mock_client_filter
    assert instance.client_side_filter_func == mock_client_filter
```

Then we can add validation later, e.g.

```
def test_server_side_filter_func_takes_iterable():
    mocked_server_filter = NonCallableMock()
    # Intentionally don't pack this into a list
    with self.assertRaises(...):
         self.instance.server_filter_funcs = mocked_server_filter
```